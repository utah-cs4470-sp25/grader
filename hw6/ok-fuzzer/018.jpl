struct a {
a : void
}
show (((((- (- 18.66)) > 48.33) == (if (! true) then (! (27.16 > 40.47)) else (! true))) || (882 < (if (false && (97.78 >= 44.3)) then (- (- 525)) else (if (if true then false else false) then 592 else (- 846))))) == (((43 - (- (- 895))) == (((341 / 17) * (- 409)) + (- (696 / 833)))) && ((- ((21.98 % 67.28) + 8.34)) == (if ((! false) && true) then (- 33.97) else (if false then 78.46 else (if false then 94.76 else 77.22))))))

struct b {
a : bool
b : float
}
show [false, (! false)]

show 2.09

struct c {
a : float
b : bool
}
show 143

show rgba {(- 28.15), c {c {(- 24.3), false}.a, (false != true)}.a, (- (if (873 == (- (if false then 466 else 177))) then 4.97 else (- 6.44))), (- 33.98)}

struct d {
a : void[]
b : b
c : rgba
}
struct e {
a : c
b : float[]
}
struct f {
a : bool
b : float
}
show (- 257)

show false

show b {((962 / (172 - (898 / 926))) > (- (- 923))), b {(false || (616 == 471)), 19.77}.b}.b

struct g {
a : void
b : a
}
show f {(50 == (- (if true then 581 else 763))), 11.16}

struct h {
a : void
}
show (84 != (if (! b {false, 19.07}.a) then [835, 632][(137 * 886)] else (- ((- (155 + 856)) + (- 6)))))

