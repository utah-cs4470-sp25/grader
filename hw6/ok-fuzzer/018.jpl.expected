(StructCmd a a (VoidType))
(ShowCmd (BinopExpr (BoolType) (BinopExpr (BoolType) (BinopExpr (BoolType) (BinopExpr (BoolType) (UnopExpr (FloatType) - (UnopExpr (FloatType) - (FloatExpr (FloatType) 18))) > (FloatExpr (FloatType) 48)) == (IfExpr (BoolType) (UnopExpr (BoolType) ! (TrueExpr (BoolType))) (UnopExpr (BoolType) ! (BinopExpr (BoolType) (FloatExpr (FloatType) 27) > (FloatExpr (FloatType) 40))) (UnopExpr (BoolType) ! (TrueExpr (BoolType))))) || (BinopExpr (BoolType) (IntExpr (IntType) 882) < (IfExpr (IntType) (BinopExpr (BoolType) (FalseExpr (BoolType)) && (BinopExpr (BoolType) (FloatExpr (FloatType) 97) >= (FloatExpr (FloatType) 44))) (UnopExpr (IntType) - (UnopExpr (IntType) - (IntExpr (IntType) 525))) (IfExpr (IntType) (IfExpr (BoolType) (TrueExpr (BoolType)) (FalseExpr (BoolType)) (FalseExpr (BoolType))) (IntExpr (IntType) 592) (UnopExpr (IntType) - (IntExpr (IntType) 846)))))) == (BinopExpr (BoolType) (BinopExpr (BoolType) (BinopExpr (IntType) (IntExpr (IntType) 43) - (UnopExpr (IntType) - (UnopExpr (IntType) - (IntExpr (IntType) 895)))) == (BinopExpr (IntType) (BinopExpr (IntType) (BinopExpr (IntType) (IntExpr (IntType) 341) / (IntExpr (IntType) 17)) * (UnopExpr (IntType) - (IntExpr (IntType) 409))) + (UnopExpr (IntType) - (BinopExpr (IntType) (IntExpr (IntType) 696) / (IntExpr (IntType) 833))))) && (BinopExpr (BoolType) (UnopExpr (FloatType) - (BinopExpr (FloatType) (BinopExpr (FloatType) (FloatExpr (FloatType) 21) % (FloatExpr (FloatType) 67)) + (FloatExpr (FloatType) 8))) == (IfExpr (FloatType) (BinopExpr (BoolType) (UnopExpr (BoolType) ! (FalseExpr (BoolType))) && (TrueExpr (BoolType))) (UnopExpr (FloatType) - (FloatExpr (FloatType) 33)) (IfExpr (FloatType) (FalseExpr (BoolType)) (FloatExpr (FloatType) 78) (IfExpr (FloatType) (FalseExpr (BoolType)) (FloatExpr (FloatType) 94) (FloatExpr (FloatType) 77))))))))
(StructCmd b a (BoolType) b (FloatType))
(ShowCmd (ArrayLiteralExpr (ArrayType (BoolType) 1) (FalseExpr (BoolType)) (UnopExpr (BoolType) ! (FalseExpr (BoolType)))))
(ShowCmd (FloatExpr (FloatType) 2))
(StructCmd c a (FloatType) b (BoolType))
(ShowCmd (IntExpr (IntType) 143))
(ShowCmd (StructLiteralExpr (StructType rgba) rgba (UnopExpr (FloatType) - (FloatExpr (FloatType) 28)) (DotExpr (FloatType) (StructLiteralExpr (StructType c) c (DotExpr (FloatType) (StructLiteralExpr (StructType c) c (UnopExpr (FloatType) - (FloatExpr (FloatType) 24)) (FalseExpr (BoolType))) a) (BinopExpr (BoolType) (FalseExpr (BoolType)) != (TrueExpr (BoolType)))) a) (UnopExpr (FloatType) - (IfExpr (FloatType) (BinopExpr (BoolType) (IntExpr (IntType) 873) == (UnopExpr (IntType) - (IfExpr (IntType) (FalseExpr (BoolType)) (IntExpr (IntType) 466) (IntExpr (IntType) 177)))) (FloatExpr (FloatType) 4) (UnopExpr (FloatType) - (FloatExpr (FloatType) 6)))) (UnopExpr (FloatType) - (FloatExpr (FloatType) 33))))
(StructCmd d a (ArrayType (VoidType) 1) b (StructType b) c (StructType rgba))
(StructCmd e a (StructType c) b (ArrayType (FloatType) 1))
(StructCmd f a (BoolType) b (FloatType))
(ShowCmd (UnopExpr (IntType) - (IntExpr (IntType) 257)))
(ShowCmd (FalseExpr (BoolType)))
(ShowCmd (DotExpr (FloatType) (StructLiteralExpr (StructType b) b (BinopExpr (BoolType) (BinopExpr (IntType) (IntExpr (IntType) 962) / (BinopExpr (IntType) (IntExpr (IntType) 172) - (BinopExpr (IntType) (IntExpr (IntType) 898) / (IntExpr (IntType) 926)))) > (UnopExpr (IntType) - (UnopExpr (IntType) - (IntExpr (IntType) 923)))) (DotExpr (FloatType) (StructLiteralExpr (StructType b) b (BinopExpr (BoolType) (FalseExpr (BoolType)) || (BinopExpr (BoolType) (IntExpr (IntType) 616) == (IntExpr (IntType) 471))) (FloatExpr (FloatType) 19)) b)) b))
(StructCmd g a (VoidType) b (StructType a))
(ShowCmd (StructLiteralExpr (StructType f) f (BinopExpr (BoolType) (IntExpr (IntType) 50) == (UnopExpr (IntType) - (IfExpr (IntType) (TrueExpr (BoolType)) (IntExpr (IntType) 581) (IntExpr (IntType) 763)))) (FloatExpr (FloatType) 11)))
(StructCmd h a (VoidType))
(ShowCmd (BinopExpr (BoolType) (IntExpr (IntType) 84) != (IfExpr (IntType) (UnopExpr (BoolType) ! (DotExpr (BoolType) (StructLiteralExpr (StructType b) b (FalseExpr (BoolType)) (FloatExpr (FloatType) 19)) a)) (ArrayIndexExpr (IntType) (ArrayLiteralExpr (ArrayType (IntType) 1) (IntExpr (IntType) 835) (IntExpr (IntType) 632)) (BinopExpr (IntType) (IntExpr (IntType) 137) * (IntExpr (IntType) 886))) (UnopExpr (IntType) - (BinopExpr (IntType) (UnopExpr (IntType) - (BinopExpr (IntType) (IntExpr (IntType) 155) + (IntExpr (IntType) 856))) + (UnopExpr (IntType) - (IntExpr (IntType) 6)))))))
